Automattic/jetpack
# Jetpack Forms: Add tracking of Google Sheets exports of responses

Dupe of #29177

When responses are exported from WP-Admin->Feedback->Form Responses they can be exported to Google Sheets. This patch adds tracking of those actions.

It requires the record_tracks_event function from https://github.com/Automattic/jetpack/pull/29102.

Discussion: pcsBup-S8-p2
Testing instructions:

Wait for https://github.com/Automattic/jetpack/pull/29102 to be merged or grab record_tracks_event from it.
Create a form block in a post with /form.
Fill out the form and submit.
Go to WP-Admin->Feedback->Form Responses and export the response(s) to Google Sheets. You'll have to connect it first. If using Firefox containers you might have to do this bit in Chrome.
If an Automattician, look for the forms_export_responses tracking event. The format prop should be "gsheets"
Does this pull request change what data or activity we track or use?

Yes, it tracks exports by Google Sheets from the Feedback Forms page.

Have you written new tests for your changes, if applicable?
Have you checked the E2E test CI results, and verified that your changes do not break them?
Have you tested your changes on WordPress.com, if applicable (if so, you'll see a generated comment below with a script to run)?
